### PR TITLE
pypi.python.org/pypi?:action=purge requires a GET

### PR DESCRIPTION
**Pull request by ewdurbin**
Should address https://github.com/pypa/pypi-legacy/issues/739

### CODE REVIEW

**Comment by ewdurbin** [comment]
Alternative option is to update pypa/pypi-legacy to accept a POST here.

**Comment by ewdurbin** [comment]
Ah, we need this to be a `POST` to bypass CDN... need to fix in pypi-legacy :)

File: warehouse/forklift/legacy.py [hunk 1]
```diff
@@ -1059,7 +1059,7 @@ def file_upload(request):
 
 def _legacy_purge(status, *args, **kwargs):
     if status:
-        requests.post(*args, **kwargs)
+        requests.get(*args, **kwargs)
```
Comment (di) [inline warehouse/forklift/legacy.py]: I think this needs to remain a `POST`? I'm not familiar with the legacy API but I'm pretty sure this won't correctly send the `:action` and `project` via the `data` kwarg.

Comment (di) [reply warehouse/forklift/legacy.py]: Oops, beat me to it. 👍

Comment (ewdurbin) [reply warehouse/forklift/legacy.py]: confirmed, i did a dumb :)

**Pull request closed without merging**